x-pack/plugins/security_solution/public/timelines/components/timeline/query_bar/index.tsx

```diff
@@ -47,6 +49,47 @@ export interface QueryBarTimelineComponentProps {
   updateReduxTime: DispatchUpdateReduxTime;
 }
 
+const SearchBarContainer = styled.div`
+  /*
+  *
+  * hide search bar default filters as they are distrubing the layout  as shown below
+  *
+  * Filters are displayed with QueryBar so below is how is the layout with default filters.
+  *
+  *
+  *                    --------------------------------
+  *   -----------------|                              |------------
+  *   | DataViewPicker |        QueryBar              | Date      |
+  *   -------------------------------------------------------------
+  *                    |      Filters                 |
+  *                    --------------------------------
+  *
+  * The tree under this component makes sure that default filters are not rendered and we can saperately display
+  * them outside query component so that layout is as below:
+  *
+  *   -----------------------------------------------------------
+  *   | DataViewPicker |      QueryBar              |   Date    |
+  *   -----------------------------------------------------------
+  *   |                       Filters                           |
+  *   -----------------------------------------------------------
+  *
+  * */
+  [data-test-subj='globalQueryBar'] [data-test-subj='filter-items-group'] {
+    display: none;
+  }
+`;
+
+const SearchBarFullScreenVisibilityStyle = createGlobalStyle`
+.euiDataGrid__restrictBody {
+  .search_bar_container {
+    .headerGlobalNav,
+    .kbnQueryBar {
+      display: flex;
+    }
+  }
+}
```

Comment:
Would it be possible to apply these styles to the `QueryBar` without using global styles?

Reply:
@semd  This is done ✅